docs/development-setup.md

```diff
@@ -20,6 +20,17 @@ yarn link "@kiltprotocol/sdk-js"
 Go into your project folder and execute that second command.
 The SDK is now symlinked in your projects `node_modules` folder.
 
+## Embed bundle in HTML
```

Comment:
Development setup is only for people working on the SDK, not _with_.

Please move these instructions into the README.md